pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -415,11 +397,24 @@ func (p *AnacondaInstallerISOTree) ostreeContainerStages() []*osbuild.Stage {
 		image,
 		nil))
 
+	stages = append(stages, p.bootcInstallerKickstartStages()...)
+	return stages
+}
+
+// bootcInstallerKickstartStages sets up kickstart-related stages for Anaconda
+// ISOs that install a bootc bootable container.
+func (p *AnacondaInstallerISOTree) bootcInstallerKickstartStages() []*osbuild.Stage {
+	if p.Kickstart == nil {
+		panic(fmt.Sprintf("Kickstart options not set for %s pipeline", p.name))
```

Comment:
slightly sad but I guess there is no other way right now - maybe `fmt.Errorf()` so that we prepare for a time when have error instead of panics (but actually ignore me, that is trivial enough to change once we add more errors)

Reply:
Every other panic in the function is a string, so I'd rather keep it like this for consistency there.

Reply:
Indeed makes sense! Consistency is more important!